setup.py

```diff
@@ -81,7 +81,7 @@ def check_api(inc_dirs):
             has_cdf5 = False
```

Comment:
This line looks problematic to me. If I build against 4.4.1.1 that has CDF5, the file exists, but not the #define. So it would be erroneously false.